bin/Cargo.toml

```diff
@@ -21,39 +20,37 @@ include = [
 name = "sozu"
 
 [dependencies]
-anyhow = "1.0.42"
-mio = { version = "^0.8", features = [ "os-poll", "net" ] }
-serde = "~1.0.2"
-serde_json = "~1.0.1"
-serde_derive = "~1.0.2"
-nom = "7.0.0"
-nix  = "^0.23"
-time = "^0.3"
-clap = "^2.19.0"
-slab = "^0.4.0"
-libc = "^0.2"
-rand = "^0.8"
-tempfile = "^3.0"
-futures = "^0.3"
-lazy_static = "^1.1"
-regex = "1.0"
-jemallocator = {version = "^0.3", optional = true }
-smol = "^1"
-async-io = "^1"
-async-dup = "1.2"
-futures-lite = "^1"
-log = "^0.4"
-hex = "^0.4"
-structopt = "^0.3"
-structopt-derive = "^0.4"
-prettytable-rs = { version = "^0.8", default-features = false}
-termion = "1.5"
+anyhow = "^1.0.56"
+mio = { version = "^0.8.2", features = [ "os-poll", "net" ] }
+serde = { version = "^1.0.136", features = ["derive"] }
+serde_json = "^1.0.79"
+nom = "^7.1.1"
+nix  = "^0.23.1"
+time = "^0.3.9"
+clap = "^2.34.0"
+slab = "^0.4.6"
+libc = "^0.2.124"
+rand = "^0.8.5"
+tempfile = "^3.3.0"
+futures = "^0.3.21"
+lazy_static = "^1.4.0"
+regex = "^1.5.5"
+jemallocator = { version = "^0.3.2", optional = true }
+smol = "^1.2.5"
+async-io = "^1.6.0"
+async-dup = "^1.2.2"
+futures-lite = "^1.12.0"
+log = "^0.4.16"
+hex = "^0.4.3"
+structopt = "^0.3.26"
+prettytable-rs = { version = "^0.8.0", default-features = false }
+termion = "^1.5.6"
 
-sozu-lib = { version = "^0.13.0", path = "../lib"  }
-sozu-command-lib = { version = "^0.13.0", path = "../command" }
+sozu-lib = { path = "../lib"  }
+sozu-command-lib = { path = "../command" }
```

Comment:
This looks fine. I'm gonna nitpick here and suggest you sort the dependencies alphabetically.